app/code/Tasks/FeedbackManager/Controller/Manage/Approvedfeedback.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace Tasks\FeedbackManager\Controller\Manage;
+
+use Magento\Framework\App\Action\Action;
+use Magento\Framework\App\Action\Context;
+use Magento\Framework\App\Config\ScopeConfigInterface;
+use Magento\Framework\App\ResponseInterface;
+use Magento\Framework\Controller\Result\JsonFactory;
+use Tasks\FeedbackManager\Model\Feedback;
+
+class ApprovedFeedback extends Action
```

Comment:
@gayathri-aravind Pls run your code through magento coding standard tools. I see few deprecated classes used here

Reply:
Run the code by installing  Magento Coding Standard(https://github.com/magento/magento-coding-standard) and phpcs. And, fixed the deprecated methods in the controllers. Commit:882571dc9e3988e3e1f109a2d06fe316213a2971